<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpaqValidation</class>
 <widget class="QWidget" name="OpaqValidation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="1">
        <widget class="QListWidget" name="modelsListWidget"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="modelsLabel">
         <property name="text">
          <string>Models:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QDateEdit" name="toDateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="toLabel">
         <property name="text">
          <string>To:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDateEdit" name="fromDateEdit">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="fromLabel">
         <property name="text">
          <string>From:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="forecastHorizonComboBox">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>Day+1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Day+2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Day+3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Day+4</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="forecastHorizonLabel">
         <property name="text">
          <string>Forecast horizon:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="aggregationComboBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="aggregationLabel">
         <property name="text">
          <string>Aggregation:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="pollutantComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="pollutantLabel">
         <property name="text">
          <string>Pollutant:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="stationComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="stationLabel">
         <property name="text">
          <string>Station:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QPushButton" name="validateButton">
         <property name="text">
          <string>Validate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
